packages/blade/src/components/Badges/_decisions/decisions.md

```diff
@@ -0,0 +1,60 @@
+# Badge <!-- omit in toc -->
+Badges are used to show small amount of color coded metadata, which are ideal for getting users attention.
+
+<img src="./badge-thumbnail.png" width="380" />
+
+## Design
+- [Figma - Badge](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=8110%3A417)
+
+
+## API
+
+| Prop | Type | Default | Description | Required |
+|---|---|---|---|---|
+| children | `string` | `undefined` | The label to be rendered within the badge. | ✅ |
+| variant | `positive`,`negative`, `notice`, `information`, `neutral` | `neutral` | The variant of the badge to be rendered. |  |
+| contrast | `low`, `high` | `low` | The contrast of the badge to be rendered. |  |
+| size | `medium`, `small` | `medium` | The size of the badge to be rendered. |  |
+| icon | `Icon` | `undefined` | The Blade `Icon` component to be rendered for the badge. |  |
+| labelStyle | `italic`, `normal` | `normal` | The font style of the badge's label. |  |
+| labelWeight | `bold`, `regular` | `regular` | The font weight of the badge's label. |  |
+
+### Sample Usage
+```jsx
+import { Badge } from '@razorpay/components';
+
+<Badge 
+  variant='positive'
+  contrast='high'
+  icon={InfoIcon} 
+  size='large' 
+  labelStyle='italic' 
+  labelWeight='bold'
```

Comment:
Isn't the code getting autoformatted to use double quotes?

Reply:
It's not 😅